src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    if (!style.length || style.indexOf(':') === -1) {
+      return acc;
+    }
+
+    const [property, value] = style.split(':');
+
+    if (property.length > 0) {
+      Object.assign(acc, { [property.trim()]: value.trim() });
```

Comment:
When splitting the style into property and value, you should also consider cases where the property or value contains a colon. For example, a style like 'font-family: "Courier New", monospace;' would not be handled correctly. You might want to split only on the first colon to avoid this issue.